docs/guide/migrating-chrony.md

```diff
@@ -0,0 +1,154 @@
+
+When doing this, it is also important to note a major philosophical difference between chrony and ntpd-rs. For chrony, the majority of the algorithm tuning parameters are set on an individual time source. Within ntpd-rs, all control of the filtering is done via global parameters. Although we do not expect this to be the case, should there be specific parameters you would wish to configure on a per-peer basis, please let us know so we can consider this for future releases.
+
+## Server configuration
+
+Server configuration in ntpd-rs works quite a bit differently from chrony. Rather than enabling time server functionality by `allow`ing remote connections to the server, one or more serving instances can be individually configured. Each of these comes with its own allow and deny list:
+```
+[[server]]
+listen="<ip or [::]>:<port>"
+allowlist = [
+    <subnet1>,
+    <subnet2>
+]
+allowlist-action = `ignore`
+denylist = [
+    <subnet3>,
+    <subnet4>
+]
+denylist-action = `deny`
+```
+The allow and deny list configuration is optional in ntpd-rs. By default, if a server is configured it will accept traffic from anywhere. When configuring both allow and deny lists, ntpd-rs will first check if a remote is on the deny list. Only if this is not the case will the allow list be considered. This needs to be taken into account when translating interleaved combinations of chrony's `allow` and `deny` commands.
+
+NTS can be enabled for a server by simply configuring an NTS key exchange server:
+```
+[[nts-ke]]
+key-exchange-listen = "<ip or [::]>:<
+certificate-chain-path = <ntsservercert>
+private-key-path = <ntsserverkey>
+```
+Here the names of the corresponding chrony directives are used on the right hand side of the assignment. Note that unlike chrony, ntpd-rs does not have a default ip address on which it listens for nts-ke traffic, and this needs to be explicitly provided. The port is optional however and defaults to the standard value 4460.
+
+The keys used to encrypt the cookies are by default ephemeral. If these should be kept across reboots of the server, the path for storing these can be configured:
+```
+[keyset]
+key-storage-path = <path to key storage>
+```
+Note that in contrast to chrony's `ntsdumpdir` directive, here the full path needs to be specified, and there is no default path.
+
+Rotation of the keys is by default daily, with one full weeks worth of old keys remembered for serving clients with older cookies. This can be configured with the `key-rotation-interval` and `stale-key-count` parameters:
+```
+[keyset]
+stale-key-count = <number of old keys to keep>
+key-rotation-interval = <rotation interval in seconds>
+```
+Note that the defaults for these settings mean that, in contrast to the chrony defaults, cookies for the server are only valid for slightly more than 1 week instead of 3.
+
+Sharing the keys with which the nts cookies are encrypted between multiple ntpd-rs servers is not yet supported.
```

Comment:
Sharing the keys, with which the NTS cookies are encrypted between multiple ntpd-rs servers, is not yet supported.